src/pages/SignIn/Index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { Button, Typography, Modal } from "antd";
+import { FieldValues, useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useNavigate } from "react-router-dom";
+import Field from "components/DefaultField/Index";
+import { useTranslation } from "react-i18next";
+import { PRIMARY } from "utils/colorConsts";
+import { PROFILE_ROUTE, SIGN_UP_ROUTE } from "utils/routeConsts";
+import S from "./style";
+import signInSchema from "./schema";
+
+interface SignInForm extends FieldValues {
+  email: string;
+  password: string;
+}
+
+export default function SignIn() {
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { Title, Text } = Typography;
```

Comment:
better move it before component, as now initialization will be made with each component rerender